en/view/category.php

```diff
@@ -0,0 +1,278 @@
+                                        <th width="10%"><small><span class="glyphicon glyphicon-sort"></span></small>Size</th>
+                                        <th width="10%"><small><span class="glyphicon glyphicon-sort"></span></small>Age</th>
+                                        <th width="10%"><small><span class="glyphicon glyphicon-sort"></span></small>Seeds</th>
+                                        <th width="10%"><small><span class="glyphicon glyphicon-sort"></span></small>Leech</th>
+                                        <th width="10%">Download</th>
+                                    </tr>
+                                  </thead>
+                                  <tbody>';
+                        foreach ($torrents as $key[] => $row) {
+                            $ymd = new DateTime($row["uploaddate"]); $today = new DateTime(); $diff=date_diff($ymd,$today);
+                            if($row["categoryname"] == $cat["categoryname"]) {
+                                echo '<tr>
+                                    <td class="Name" data-label="Name"><a href="torrent.php?hash='.$row["hash"].'&id='.$row["userid"].'">'.$row["name"].'</a>
+                                        <small>by <a href="user-torrents.php?userid='.$row["userid"].'">'.$row["username"].'</a></small>
+                                    </td>
+                                    <td data-label="Size">'.$row["size"].'</td>
+                                    <td data-label="Age">'. $diff->format("%ad").'</td>
+                                    <td data-label="Seeds">'.$row["seeders"].'</td>
+                                    <td data-label="Leech">'.$row["leechers"].'</td>
+                                    <td data-label="Download">                                        
+                                        <a href="'.$row["magnet"].'"><span class="glyphicon glyphicon-magnet link"></span></a>
+                                    </td>
+                                </tr>';
+                            }
+                        }
+                        echo '</tbody></table>';
+                    }
+                ?>
+            </div>
+        </div>
+            <!-- /.row -->
+			<?php if($total_torent_page>0){ ?>
+<nav aria-label="Page navigation example">
+  <ul class="pagination">
+			<?php if($current_page>1){ ?>
+    <li class="page-item"><a class="page-link" style="float: left;" href="?id=<?=$cat_id?>&page=<?=$current_page-1?>">Previous</a></li>
+   
+			<?php } if($current_page<$total_torent_page){ ?> <li class="page-item"><a class="page-link" style="    float: right;" href="?id=<?=$cat_id?>&page=<?=$current_page+1?>">Next</a></li>
+  
+			<?php } ?></ul>
+</nav>
+			<?php } ?>
+        <hr>
+
+        <!-- Footer -->
+        <footer>
+            <div class="row">
+                <div class="col-lg-12">
+                    <p>Happy Downloading~</p>

```

Comment:
Copyright &copy; Your Website 2019

Reply:
Changed 'Happy Downloading~' to 'Copyright © Your Website 2019'